container/common/helpers.go

```diff
@@ -226,7 +203,10 @@ func readString(dirpath string, file string) string {
 
 func readUInt64(dirpath string, file string) uint64 {
 	out := readString(dirpath, file)
-	if out == "" || out == "max" {
+	if out == "max" {
```

Comment:
Why have you decided to change behaviour for `out == ""`?

Reply:
for `out == ""` I've left the same behavior there was before 2ccad4b42fe52b312f6d75a312c61a54fd123dc5

Reply:
Of course, my bad!